<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>debuggerPage</class>
 <widget class="QWidget" name="debuggerPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>501</width>
     <height>521</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,2">
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="rowCount">
       <number>12</number>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column>
       <property name="text">
        <string>变量名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>值</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="runBtn">
        <property name="text">
         <string>run</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="continueBtn">
        <property name="text">
         <string>continue</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="setEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="setBtn">
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="deleteEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="deleteBtn">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Start Debugging</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>End</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
